manifests/h/HHDSoftwareLtd/VirtualSerialPortTools/5.01.02.0992/HHDSoftwareLtd.VirtualSerialPortTools.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: HHDSoftwareLtd.VirtualSerialPortTools
+PackageVersion: 5.01.02.0992
+PackageLocale: en-US
+Publisher: HHD Software Ltd.

```

```suggestion
Publisher: HHD Software, Ltd.
```
to match Control Panel.